### PR TITLE
Playa fireworks

### PR DESCRIPTION
**Pull request by ericrius1**
This PR provides a script which spawns a launchpad from which one can launch fireworks

### CODE REVIEW

**Comment by ericrius1** [comment]
test script: https://rawgit.com/ericrius1/hifi/playaFireworks/examples/playa/fireworks/fireworksLaunchButtonSpawner.js

asana: https://app.asana.com/0/31759584831097/93215362870208

To Use:

Run the above test script-
1) You should see a bug red launch pad appear in front of you.
2) In that entity's userData, set the launchPosition to the position you would like the fireworks to launch from
3) Now mouse click or, with hydras, near trigger the launch pad, and watch the show

**Comment by hifi-gustavo** [comment]
Build console output for commit: ad49bcda72cd97df7049d923fb9a9413f5ff6524 available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by cozza13** [comment]
that is fun!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8f914c6673ed105339aadb9f3fecdeaf30a2bb5b available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 728612f9a7892b3e8e70333a52200ef940123f19 available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9a108c298fb77e687f77860785cfac18287572c0 available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d4d5aed51867f7b6cee3a60a3ff495774043211e available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f12acfb7f3e64a1741eab47579ce44b49c4c587e available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 54ee3d347c2098237146e4086e52727523eadc6d available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 98cf784c6f2aba50a4a44819c923846652da7dfd available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

File: examples/playa/fireworks/fireworksLaunchButtonEntityScript.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
+  //
+  //  fireworksLaunchEntityScript.js
+  //  examples/playa/fireworks
+  //
+  //  Created by Eric Levina on 2/24/16.
```
Comment (imgntn) [inline examples/playa/fireworks/fireworksLaunchButtonEntityScript.js]: you spelled your name wrong and the copyright should be 2016 ;)

**Comment by imgntn** [comment]
@ericrius1 there are a lot of magic numbers in this code -- do you think making some of them constants will make it easier to play with?  i don't personally think its a blocker but coding standard

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4d27d5c9f2787e0d0051ee934b5632101cafa1bc available at: https://gist.github.com/5546d3631061292bdb0f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7190.dmg

**Pull request merged**